ValueError: В строке 1 число